### ISSUE TITLE
"View slides" button has insufficient contrast

### ISSUE THREAD

**Issue by phette23** [code4lib/2021.code4lib.org#198]
<img width="502" alt="Screen Shot 2021-03-17 at 8 37 02 PM" src="https://user-images.githubusercontent.com/1024833/111569789-a2a0f600-8760-11eb-8618-b7144de513a8.png">

I noticed this on [my workshop page](https://2021.code4lib.org/workshops/Learn-to-Love-the-Command-Line) and indeed the LIghthouse tools flag it. Probably hasn't come up simply because slides are added on most posts.

Great work on the website this year! It looks amazing!

**Comment by dzoladz** [comment]
Thanks for the heads-up. I hadn't noticed a button for slides on individual talks. I'll fix this in just a moment.
